File: JavaScript/1920. Build Array from Permutation.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var buildArray = function(nums) {
    var a = [];
    for(var i = 0;i < nums.length;i++){
        a[i] = nums[nums[i]];
    }
    return a;
};

File: Java/1920. Build Array from Permutation.java
class Solution {
    public int[] buildArray(int[] nums) {
        int l = nums.length;
        int[] ans = new int[l];

        for(int i = 0; i < l; i++){
            ans[i] = nums[nums[i]];
        }

        return ans;
    }
}
